doc/installing-a-development-environment.md

```diff
@@ -1,5 +1,14 @@
 ## Installing
 
+### Introduction to Installation
+
+The development environment setup requires installation of following softwares:
+
+1. Ruby - Needs to be installed in order to install and use Jekyll. Jekyll helps in tranforming plain texts into static websites.
```

Comment:
How about: "Needed by Jekyll. Jekyll is the static site generator that generates the website and docs."

Reply:
@dblotsky  sounds good to me. Will make this change.